docs/refguide/configure.md

```diff
@@ -0,0 +1,212 @@
+---
+title: Configuring MobX
+sidebar_label: Configuring MobX 🚀
+hide_title: true
+---
+
+<script async type="text/javascript" src="//cdn.carbonads.com/carbon.js?serve=CEBD4KQ7&placement=mobxjsorg" id="_carbonads_js"></script>
+
+# Configuring MobX
+
+MobX has several configuration depending on how you prefer to use MobX, which JavaScript engines you want to target, and whether you want MobX to hint at best practices.
+Most configuration options can be set by using the `configure` method.
+
+## Proxy support
+
+By default MobX uses proxies to make arrays and plain objects observable. Proxies provide the best performance and most concistent behavior accross environments.
+However, if you are targetting an environment that doesn't support proxies, proxy support has to be disabled.
+Most notably this is the case when targetting Internet Explorer or React Native without using the Hermes engine.
+
+Proxy support can be disabled by using `configure`:
+
+```typescript
+import { configure } from "mobx"
+
+configure({
+    useProxies: "never"
+})
+```
+
+Accepted values for the `useProxies` configuration are:
+
+-   `"always"` (default): MobX is expected to only run in environments with [`Proxy` support](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy). MobX will error if these are not available.
+-   `"never"`: Proxies are never used. MobX falls back on non-proxy alternatives. This is compatible with all ES5 environments, but causes various [limitations](#limitations-without-proxies).
+-   `"ifavailable"`: (Experimental option) Proxies are used if they are available, and otherwise MobX falls back to non-proxy alternatives. The benefit of this mode is that MobX will try to warn if API's or language features that wouldn't work in ES5 enviroments are used, triggering errors when hitting an ES5 limitation when running on a modern environment.
```

Comment:
```suggestion
-   `"ifavailable"`: (Experimental option) Proxies are used if they are available, and otherwise MobX falls back to non-proxy alternatives. The benefit of this mode is that MobX will try to warn if APIs or language features that wouldn't work in ES5 environments are used, triggering errors when hitting an ES5 limitation when running on a modern environment.
```